README.md

```diff
@@ -0,0 +1,189 @@
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [JavaScript capstone project] <a name="about-project"></a>
+
+**[JavaScript capstone project]** is about building your own web application based on an external API
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+     <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A home page showing a list of items]**
+- **[A popup window with more data about an item]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://henarykhuraijam.github.io/kanbanBoard/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+
+-Visual Studio Code or any other code editor of your preference
+
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+cd kanbanBoard
+git https://github.com/HenaryKhuraijam/kanbanBoard.git
+```
+
+### Install
+Install this project with:
+
+```sh
+npm init -y
+npm install --save-dev hint@7.x
+```
+
+### Run tests
+To track linter errors locally follow these steps:
+
+Download all the dependencies run:
+```sh
+npm install --save-dev hint@7.x
+```
+
+Track HTML linter errors run:
+```sh
+npx hint .
+```
+
+Track CSS linter errors run:
+```sh
+npx stylelint "**/*.{css,scss}"
+```
+Track Javascript linter errors run:
+```sh
+npx eslint .
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
```

Comment:
- [ ] As I mentioned already in my above two comments about professionalism, those things apply here and I am writing it again here, **always add the relevant things to your project documentation**. As you can see you have added something which have nothing to do with your project. It can put someone into some confusion because `gem` is part of `Ruby and its ecosystem` and not meant for `JavaScript` based projects at this point.

Reply:
Done